plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/GroupedProductSelector.php

```diff
@@ -32,9 +32,31 @@ protected function render( $attributes, $content, $block ): string {
 		global $product;
 
 		if ( $product instanceof \WC_Product && $product->is_type( 'grouped' ) ) {
+			add_filter( 'woocommerce_product_supports', array( $this, 'check_product_supports_ajax_add_to_cart' ), 10, 3 );
```

Comment:
@mikachan heads-up that https://github.com/woocommerce/woocommerce/pull/58588 has been merged. So now we will need to remove this hook.

Reply:
Thanks for the heads-up! I've removed that hook in https://github.com/woocommerce/woocommerce/pull/58002/commits/f92d03d7ff4bb294d9ed74fb82a3dde26bdf3b21.